2_process/out/model_diagnostic_text.txt

```diff
@@ -0,0 +1 @@
+resulted in mean RMSEs (means calculated as average of RMSEs from the five dataset iterations) of 0.92, 1.07, and 1.56°C for the PGDL, DL, and PB models, respectively.	 The relative performance of DL vs PB depended on the amount of training data. The accuracy of Lake Mendota temperature predictions from the DL was better than PB when trained on 500 profiles 	 (1.28 and 1.49°C, respectively) or more, but worse than PB when training was reduced to 100 profiles (1.73 and 1.64°C respectively) or fewer.	 The PGDL prediction accuracy was more robust compared to PB when only two profiles were provided for training (1.77 and 3.03°C, respectively). 
```

Comment:
As with the 1_fetch/out folder, contents of 2_process/out can be deleted and .gitignored.

Reply:
Thanks, and done.